esmvaltool/recipes/ref/recipe_ref_annual_cycle_region.yml

```
@@ -0,0 +1,149 @@
+# ESMValTool
+---
+documentation:
+  title: Benchmarking of a single model for a region through extract_shape.
+  description: >
+    Benchmarking: annual cycle.
+  authors:
+    - lauer_axel
+    - bock_lisa
+    - hassler_birgit
+    - lindenlaub_lukas
+    - schlund_manuel
+    - weigel_katja
+  maintainer:
+    - lauer_axel
+  references:
+    - lauer25gmd
+  projects:
+    - dlrmabak
+
+
+datasets:
+  # Dataset to be benchmarked
+  - {dataset: MIROC6, grid: gn, benchmark_dataset: true, plot_label: 'MIROC6'}
+
+preprocessors:
+  pp_abs:
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    climate_statistics:
+      period: monthly
+    area_statistics:
+      operator: mean
+
+  pp_abs_200:
+    extract_levels:
+      levels: 20000
+      scheme: linear
+      coordinate: air_pressure
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    climate_statistics:
+      period: monthly
+    area_statistics:
+      operator: mean
+
+
+diagnostics:
+  anncyc:
+    description: Plot global mean mean annual cycle for the given period including a reference dataset.
+    variables:
+      tas: &var_default_abs
+        start_year: 1980
+        end_year: 2009
+        preprocessor: pp_abs
+        project: CMIP6
+        mip: Amon
+        exp: historical
+        ensemble: r1i1p1f1
+        grid: gr
+        reference_dataset: HadCRUT5
+        additional_datasets:
+          - {dataset: HadCRUT5, project: OBS, type: ground, version: 5.0.1.0-analysis,
+             tier: 2, reference_for_metric: true'}
+      pr:
+        <<: *var_default_abs
+        reference_dataset: GPCP-SG
+        additional_datasets:
+          - {dataset: GPCP-SG, project: obs4MIPs, type: atmos, version: 2.3, tier: 1,
+             reference_for_metric: true'}
+      psl:
+        <<: *var_default_abs
+        reference_dataset: ERA-5
+        additional_datasets:
+          - {dataset: ERA-5, project: obs4MIPs, type: atmos, version: PCMDI, tier: 1,
```

```suggestion
          - {dataset: ERA-5, project: obs4MIPs, tier: 1,
```
`type` is not needed for obs4MIPs data and the only available version is `v20250220`, but the version does not need to be specified because ESMValCore will automatically use the latest available version.